.changeset/sixty-chefs-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+`IconButton`s inside a vertical `ButtonGroup` will show tooltips on the right side by default, to avoid obscuring adjacent buttons in the group. This can be customized using `labelProps.placement`.
```

Comment:
Should this also mention that `labelProps` is a new prop added to `IconButton`?

Reply:
something like this?
```suggestion
`IconButton`s inside a vertical `ButtonGroup` will now show tooltips on the right side by default, to avoid obscuring adjacent buttons in the group. The behavior of these tooltips can also be customized using the new `labelProps` prop, which can contain `labelProps.placement`, etc.
```

Reply:
Yes, sounds good! 👍